Makefile

```diff
@@ -298,8 +298,9 @@ endif
 ifeq ($(OS),WINNT)
 define std_dll
 julia-deps: | $$(build_bindir)/lib$(1).dll
-$$(build_bindir)/lib$(1).dll: | $$(build_bindir)
-	cp $$(call pathsearch,lib$(1).dll,$$(STD_LIB_PATH)) $$(build_bindir) ;
+$$(build_bindir)/lib$(1).dll: | $$(build_bindir) $$(build_depsbindir)
```

Comment:
a target should only do exactly one thing, not make two files. make this two targets and add both to the julia-deps target on line 300